pages/common/meshlabserver.md

```diff
@@ -0,0 +1,23 @@
+# meshlabserver
+
+> Command line interface for the MeshLab 3D mesh processing software.
+
+- Convert an STL file to an OBJ file:
+
+`meshlabserver -i {{input.stl}} -o {{output.obj}}`
+
+- Convert a WRL file to a OFF file, inlcuding the vertex and face normals in the output:
```

Comment:
Huh. It seems like this is actually `-o -m`, at least if I'm reading [this](https://github.com/svn2github/meshlab/blob/master/meshlab/src/meshlabserver/meshlabserver.txt#L16-L27) right. I'll make some tests later and update accordingly :)